lib/tasks/solid_errors_tasks.rake

```diff
@@ -0,0 +1,15 @@
+namespace :solid_errors do
+  desc "Copy and run migrations from Solid Errors"
+  task install_migrations: :environment do
+    source = File.expand_path("../../solid_errors/db/migrate", __FILE__)
+    destination = File.join(Rails.root, "db", "migrate")
+
+    Dir.glob("#{source}/*.rb").each do |migration|
+      filename = File.basename(migration)
+      unless File.exist?(File.join(destination, filename))
+        puts "Copying #{filename} from Solid Errors to Rails application"
+        FileUtils.cp(migration, destination)
```

Comment:
I think the timestamp in the file name should be now, not when we create the file. If we use the timestamp we created the file, it could be inserted deep in the past of the migrations of the app, even tho it will be run now.

Reply:
Ok. Just committed these two changes (the `.one?` above) and this one. My one concern with making the migration's timestamp `Time.now` is that I was worried it could copy the migration over multiple times whenever a user run the rake task command.

Reply:
Check for the existence of the file name within the timestamp. You can use a glob for that

Reply:
```
unless File.exist?(File.join(destination, filename_with_timestamp)) && Dir.glob("#{destination}/**/*#{original_filename}*").empty?
        puts "Copying #{original_filename} to #{filename_with_timestamp} from Solid Errors to Rails application"
        FileUtils.cp(migration, File.join(destination, filename_with_timestamp))
      end
```

I tend to be explicit at first and when I look at this, not seeing a reason to keep the File.exist?. I have a hard time imagining that any timestamp would be identical. Seems like we could just go with the Dir.glob?

Reply:
I think the only thing we need to check is that we don't copy the same migration multiple times into the app. If we add a new migration later and the user runs the Rake task again, we should only copy migrations that don't yet exist in the app

Reply:
Yep. Sounds good. Updated that rake task in the last commit to handle that requirement.